docs/.gitignore

```diff
@@ -1,2 +1,3 @@
 public/
+templates/swagger/v1_json.tmpl
```

Comment:
The file should not be ignored as CI check that the file is valid with the generation (re-generate and check if git detect changes)

Reply:
It was ignored all the time. This file was moved from `public` directory, which, as you can see, is in `.gitignore` too.

Reply:
Oh sorry, I didn't see that it wasn't top-level ...

Reply:
No problem.